template.tpl

```diff
@@ -318,6 +318,43 @@ ___TEMPLATE_PARAMETERS___
         "type": "EQUALS"
       }
     ]
+  },
+  {
+    "type": "GROUP",
+    "name": "additionalInformation",
+    "groupStyle": "ZIPPY_CLOSED",
+    "subParams": [
+      {
+        "type": "SIMPLE_TABLE",
+        "name": "products",
+        "displayName": "Product Information",
+        "simpleTableColumns": [
+          {
+            "defaultValue": "",
+            "displayName": "Product ID",
+            "name": "id",
+            "type": "TEXT",
+            "valueHint": ""
+          },
+          {
+            "defaultValue": "",
+            "displayName": "Product Name",
+            "name": "name",
+            "type": "TEXT",
+            "valueHint": ""
+          },
+          {
+            "defaultValue": "",
+            "displayName": "Product Category",
+            "name": "category",
+            "type": "TEXT",
+            "valueHint": ""
```

Comment:
Do we wanna provide a hint as to what category names are acceptable?  While ultimately this'll be free form -- right? -- for the first version of shopping ads are we using the IAB content taxonomy?

Reply:
Yeah good call!